rust-sdk/whirlpool/src/decrease_liquidity.rs

```diff
@@ -27,20 +27,96 @@ use crate::{
 
 // TODO: support transfer hooks
 
+/// Represents the parameters for decreasing liquidity in a pool.
+///
+/// You must specify only one of the parameters (`TokenA`, `TokenB`, or `Liquidity`).
+/// Based on the provided value, the SDK computes the other two parameters.
+///
+/// # Variants
+///
+/// * `TokenA(u64)` - Specifies the amount of Token A to withdraw.
+/// * `TokenB(u64)` - Specifies the amount of Token B to withdraw.
+/// * `Liquidity(u128)` - Specifies the amount of liquidity to decrease.
 #[derive(Debug, Clone)]
 pub enum DecreaseLiquidityParam {
     TokenA(u64),
     TokenB(u64),
     Liquidity(u128),
 }
 
+/// Represents the instructions and quote for decreasing liquidity in a position.
+///
+/// # Fields
```

Comment:
Same here I potentially you can put the comment next to the field itself

Reply:
👌🏻

## Before
![Screenshot 2024-11-18 154537](https://github.com/user-attachments/assets/b574579e-6436-423f-9a35-7b1b210ba186)

## After
![Screenshot 2024-11-18 154611](https://github.com/user-attachments/assets/bc110368-ede5-431e-b01f-85765a686675)